src/c++/perf_analyzer/genai-pa/genai_pa/constants.py

```diff
@@ -25,3 +25,7 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 LOGGER_NAME: str = "genai-pa"
+
+OPEN_ORCA = "openorca"
+CNN_DAILY_MAIL = "cnn_dailymail"
+DEFAULT_INPUT_DATA_JSON = "./llm_inputs.json"
```

Comment:
had to remove the "./" in my branch. You will see a slight merge conflict without that update.

Reply:
Fixed to match and am now using this define inside the class